_posts/2020-09-22-data-explorer-kafka-connect.md

```diff
@@ -0,0 +1,413 @@
+
+> Resources for this blog post are [available on GitHub](https://github.com/abhirockzz/kusto-kafka-connect-strimzi)
+
+## Prerequisites
+
+You will need an [Azure account](https://azure.microsoft.com/en-us/free/) along with [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest) or [Azure Cloud Shell](https://azure.microsoft.com/features/cloud-shell/).
+
+Here are some quick pointers to setting up a Azure Data Explorer cluster and a managed Kubernetes service on Azure. I recommend installing the below services as a part of a single [Azure Resource Group](https://docs.microsoft.com/azure/azure-resource-manager/management/overview) which makes it easy to manage these services
+
+### Azure Data Explorer
+
+You can setup an Azure Data Explorer cluster and database [using Azure Portal](https://docs.microsoft.com/azure/data-explorer/create-cluster-database-portal), [Azure CLI](https://docs.microsoft.com/azure/data-explorer/create-cluster-database-cli) or any of the client SDKs such as [Python](https://docs.microsoft.com/azure/data-explorer/create-cluster-database-python). Once that's done, create a table (named `Storms`) and respective mapping (named `Storms_CSV_Mapping`) using below queries:
+
+```kusto
+.create table Storms (StartTime: datetime, EndTime: datetime, EventId: int, State: string, EventType: string, Source: string)
+
+.create table Storms ingestion csv mapping 'Storms_CSV_Mapping' '[{"Name":"StartTime","datatype":"datetime","Ordinal":0}, {"Name":"EndTime","datatype":"datetime","Ordinal":1},{"Name":"EventId","datatype":"int","Ordinal":2},{"Name":"State","datatype":"string","Ordinal":3},{"Name":"EventType","datatype":"string","Ordinal":4},{"Name":"Source","datatype":"string","Ordinal":5}]'
+```
+
+### Azure Kubernetes Service (optional)
+
+I have used [Azure Kubernetes Service](https://docs.microsoft.com/azure/aks/) (AKS) but the instructions in this blog post should work for other options as well (e.g. with a local `minikube` cluster on your laptop). You can setup an AKS cluster using [Azure CLI](https://docs.microsoft.com/azure/aks/kubernetes-walkthrough), [Azure portal](https://docs.microsoft.com/azure/aks/kubernetes-walkthrough-portal) or [ARM template](https://docs.microsoft.com/azure/aks/kubernetes-walkthrough-rm-template) 
+
+## Base installation
+
+Start by installing the Strimzi Operator and use it to spin up a single-node Kafka Cluster on Kubernetes. Installing Strimzi using `Helm` is pretty easy:
+
+```shell
+helm repo add strimzi https://strimzi.io/charts/
+helm install strimzi-kafka strimzi/strimzi-kafka-operator
+```
+
+To confirm successful installation:
+
+```
+kubectl get pods -l=name=strimzi-cluster-operator
+
+//output:
```

Comment:
```suggestion
# output:
```

Also, this is a little awkward because you're using a single code block for both input and output with only the comment to suggest what's happening. In the following code block you provide two commands separated by a comment, so it's a little inconsistent. But I'm not sure adding a shell prompt `$` would help much because it would make it slightly harder to copy and paste the commands. So maybe use separate code blocks for the input and output?

Reply:
Thanks for the suggestion. Updated the blog post to separate out the command and output